services/powchain/src/main/java/tech/pegasys/artemis/services/powchain/PowchainService.java

```diff
@@ -51,7 +53,9 @@
 import tech.pegasys.artemis.util.time.TimeProvider;
 
 public class PowchainService implements ServiceInterface {
+
   private static final Logger LOG = LogManager.getLogger();
+  public static final int MAXIMUM_CONCURRENT_REQUESTS = 5;
```

Comment:
I'd suggest setting this constant in the constants file so that configurable constants like these are more visible to outsiders and also more visible to us long after we develop this specific module.

Reply:
Done, but I think we should actually move away from that pattern.  Each thing should live in the right place not just be dumped into a catch-all class.  But there's bigger design issues at play there anyway.